src/gwt/src/org/rstudio/studio/client/workbench/views/viewer/ViewerPane.java

```diff
@@ -280,7 +280,16 @@ public void onResize()
       
       publishButton_.setShowCaption(width > 500);
    }
-   
+
+   @Override
+   public void setFocus()
```

Comment:
This pane (when nothing has been loaded in it) has no focusable controls (all the buttons are disabled). So it actually acts as a "trap" and F6 refuses to go to it, staying stuck on the previous pane.

Simplest fix might be to make the "clear" button always enabled, and just be a no-op if there's nothing to clear. Then you don't need any of this logic involving zoomButton_ or clearAllButton_ and the default behavior will handle it for you. That is, don't need a `setFocus()` override at all or the new member variables.

Reply:
Fixed!